### DISCUSSION TITLE
Crash at Apple logo

### DISCUSSION THREAD

**Discussion by tbravenec** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#38] [Q&A]
Hello,

I have a G14 2021 GA401QM (Ryzen 7 5800HS 8 cores, 16 threads, RTX3060), at first I downloaded the EFI from the repository and then I used OCAuxiliaryTools to Generate the PI. I also cleared the Nvram. I also disabled the NootedRed before trzing to install according to the readme. However when I tried to install I only see apple logo, then for few seconds loading bar followed up by a reboot of the system... Is there something I am doing wrong? 
I am sorry if this is a common issue, but this is my first time trying to install macOS.
Thank you for your answers!

**Comment by Z2VvZ2Vv** [comment]
Hey ! Put the verbose mode by going to NVRAM on OCAuxiliaryTools on the boot args just add "-v" on the UUID begin by "7C4...".

If you don't really understand, check this video : https://www.youtube.com/watch?v=QhKqq5DpoH8
Then send photos of the lines to see what can be the solutions !

**Comment by PIut02** [comment]
If you set bios correctly as described in readme, try disabling SSDT-NoHybGfx.aml and try again. There are reports that this will cause crashes but the cause is unknown.

**Comment by tbravenec** [comment]
Thank you, I enabled the verbose mode and disabled SSDT-NoHybGfx.aml, now the installer does not crash immediately, but later on, after writing unsupported CPU message... Not sure if this can be fixed...
![PXL_20240404_073032878_exported_199759](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/22779496/ca131eff-2b42-4bf6-8f73-602a758822e7)

I attached a video of the log here: https://youtu.be/TxfRKbCYA7U
I tried to record it in slowmotion, just in case it helps.

**Reply by PIut02** [reply]
First I have to ask what is your ssd model?

**Reply by tbravenec** [reply]
Samsung PM991, which is on the incompatible list, so no MacOS for me :) Thank you for your help! We can close this thread :)
